### ISSUE TITLE
How to reduce size of the model?

### ISSUE THREAD

**Issue by Rakeshvcr** [davisking/dlib-models#28]
Hello,
I have trained shape_predictor_68_face_landmarks with the params mentioned the paper. The problem is for my application i need the model size to less the 10mb. i was able to get reduce model size by keeping less params, by doing so accuracy became very low. 
Is there any way to save the model with less model size. In TensorFlow they use quantization for reducing model size, similarly is there any thing that i can follow.

(sorry for asking this kind of questions)

**Comment by davisking** [comment]
Make it output less than 68 points, or fewer trees, or trees that are less deep.
